From 01af235657e4d8f5f87b5e06bcf42baf53e1ff8d Mon Sep 17 00:00:00 2001
From: Florinel Iordache <florinel.iordache@nxp.com>
Date: Mon, 5 Nov 2018 17:03:04 +0200
Subject: [PATCH] arm64: dts: lx2160: DPMAC connections to backplane PHYs
 example

This is an example of device tree nodes required to enable 10GBase-KR and 40GBase-KR on LX2160

Signed-off-by: Florinel Iordache <florinel.iordache@nxp.com>
---
 arch/arm64/boot/dts/freescale/fsl-lx2160a-qds.dts | 20 ++++++++++++++++++++
 1 file changed, 20 insertions(+)

--- a/arch/arm64/boot/dts/freescale/fsl-lx2160a-qds.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-lx2160a-qds.dts
@@ -202,3 +202,23 @@
 		fsl,lane-reg = <0xE00 0x100>; /* lane G */
 	};
 };
+
+/* Update DPMAC connections to 40G backplane PHYs
+ * &dpmac1 {
+ *	phy-handle = <&pcs_phy1>;
+ * };
+ * 
+ * &dpmac2 {
+ *	phy-handle = <&pcs_phy2>;
+ * };
+ */
+
+/* Update DPMAC connections to 10G backplane PHYs
+ * &dpmac3 {
+ *	phy-handle = <&pcs_phy3>;
+ * };
+ * 
+ * &dpmac4 {
+ *	phy-handle = <&pcs_phy4>;
+ * };
+ */
